src/targets/browser/index.ejs

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="{{.Locale}}">
+<head>
```

Comment:
m4dz point was that browsers are smart enought to fix this point by themself.

I a pretty much no opinion so this version fits me as well.

Reply:
I confirm: the browsers are smart enough to do that.

But this trick was not used consistently: `<head>` was still present in a lot of templates in our applications, and I prefer to be explicit. So, I've put again `<head>` in our templates, even if it is not strictly needed.